merkletree/str.go

```diff
@@ -21,10 +21,10 @@ type SignedTreeRoot struct {
 	PreviousEpoch   uint64
 	PreviousSTRHash []byte
 	Signature       []byte
-	Policies        Policies
+	Policies        *Policies
```

Comment:
Sort of ... there's an expectation that policies are to be able to be altered between epochs.  With this change, we're asking developers to be aware that they need to allocate a new Policies structure each time, rather than manipulate the one they may be retaining since startup. It just feels like a copy is less error prone.

After writing that sentence, I looked at directory.go and saw that we're doing this right,
https://github.com/coniks-sys/coniks-go/blob/master/protocol/directory.go#L43
but still feel uncomfortable with the pointer.  Maybe others have an opinion?

Also, I think we have a bug #80